index.html

```diff
@@ -58,7 +58,13 @@ <h1>Lipoic</h1>
       <div class="right">
         <h1>Lipoic 來品一課</h1>
         <h2>線上整合教育平台</h2>
-        <a class="loginBtn" href="https://app.lipoic.org">登入</a>
+        <!-- <a class="loginBtn" href="https://app.lipoic.org">登入</a> -->
+        <a
+          class="loginBtn"
+          href="#"
+          onclick="alert('應用正在開發中，尚未開放註冊。若您為開發者，欲貢獻開發，請聯繫Lipoic官方(discord)')"
```

Comment:
```suggestion
          onclick="alert('應用正在開發中，尚未開放註冊。若您為開發者，欲貢獻開發，請聯繫 Lipoic 官方（Discord）。')"
```
if you want to use Chinese